library/objective-c/EnvoyHTTPStreamImpl.m

```diff
@@ -74,14 +75,15 @@
   return NULL;
 }
 
-static void *ios_on_cancel(envoy_stream_intel stream_intel, void *context) {
+static void *ios_on_send_window_available(envoy_stream_intel stream_intel, void *context) {
```

Comment:
Looks like this got switched up with `ios_on_cancel`.

Reply:
Renamed to `ios_on_cancel_impl` in 7af1f6e

Reply:
Instead, could update `on_cancel` and just ignore the new arg? And then add a no-op on_send_window_available stub.

Reply:
I was under the impression that `on_send_window_available` was calling `ios_on_cancel` on purpose.

If that can be a no-op for now, then I'll do that. I didn't want to change behavior in this PR without that context.

Reply:
nit - since you're not using the parameters, you can skip naming them and just include the type.

Reply:
This is Objective-C though, not C++/Objective-C++, so I get this error if I omit the parameter name:

```
external/envoy_mobile/library/objective-c/EnvoyHTTPStreamImpl.m:66:54: warning: omitting the parameter name in a function definition is a C2x extension [-Wc2x-extensions]
                             envoy_final_stream_intel, void *context) {
                                                     ^
```